app/root.jsx

```diff
@@ -28,7 +28,21 @@ export default function App() {
       <head>
         <meta charSet="utf-8" />
         <meta name="viewport" content="width=device-width,initial-scale=1" />
+        <meta name="title" content="Alby — Lightning buzz for your Browser!"/>
+        <meta name="description" content="Alby brings Bitcoin to the web with in-browser payments and identity."/>
```

Comment:
here we can have a longer description @MoritzKa I think we should mention lightning.